src/api_v2/directives.md

```diff
@@ -247,6 +247,7 @@ https://github.com/Normation/rudder/blob/master/rudder-web/src/main/scala/com/no
     @apiParam (Mono) {Boolean}           [enabled=true]                              Enable the Directive at creation. This parameter can be overriden. Used to be 'isEnabled' in response, look <a href="#non-compliant-data-in-response">here</a> for more details.
     @apiParam (Mono) {TechniqueVersion}  [techniqueVersion=latest Technique version] In which version your tehcnique should be created.
     @apiParam (Mono) {Int}               [priority]                                  Directive priority. Determines which value to use if there are several Directives based on the same technique applied to a Node.
+    @apiParam (Multi) {Tag}              [tags]                                      Tags linked to this directive. they have a "key:value" format.
```

Comment:
I believe it is not exactly the case. I think that at some point, we changed it to an array of ... not sur. {"name":"...", "value":...}. Due to the the client side, I think?